src/ts/is_a.ts

```diff
@@ -1,16 +1,17 @@
 
 interface typeMap { // for mapping from strings to types
-  string  : string;
-  number  : number;
-  boolean : boolean;
```

Comment:
this isn't a mistake.  horizontal alignment of colons is part of my idiomatic style